src/client/components/data-table/index.pug

```diff
@@ -37,7 +37,7 @@ table.rwd-table
         +if('hasAddNewButton')
             tr
                 td(colspan="{{ columnList.length + 1 }}")
-                    row
+                    .row
```

Comment:
they are the same as we use Ractive-bootstrap plugin's `row` component, but it doesn't hurt anyway...